### PR TITLE
use raw loader instead of requireText 

### PR DESCRIPTION
**Pull request by i-like-robots**
Fixes [this](https://financialtimes.slack.com/files/dilip.vaghela/F10M8M6US/screen_shot_2016-04-14_at_14.53.19.png)

### CODE REVIEW

File: client/components/related-content/main.js [hunk 1]
```diff
@@ -48,7 +48,7 @@ module.exports = {
 
 				new VideoStream(relatedItemsEl, {
 					items: json.items.slice(0, 2),
-					itemTpl: '<li data-trackable="related item">' + requireText('../video-stream/card.html') + '</li>',
+					itemTpl: '<li data-trackable="related item foobar">' + require('../video-stream/card.html') + '</li>',
```
Comment (ironsidevsquincy) [inline client/components/related-content/main.js]: foobar

**Comment by ironsidevsquincy** [comment]
:+1:

File: client/components/related-content/main.js (hunk 1 above)
Comment (bjfletcher) [inline client/components/related-content/main.js]: Ha ha I'm not the only one then ;)

Comment (i-like-robots) [inline client/components/related-content/main.js]: 😳

**Pull request merged**